client/asset/eth/node.go

```diff
@@ -32,8 +33,68 @@ var simnetGenesis string
 type nodeConfig struct {
 	net                dex.Network
 	listenAddr, appDir string
+	logger             dex.Logger
 }
 
+// ethLogger satisifies geth's logger interface.
+type ethLogger struct {
+	dl dex.Logger
+}
+
+// New returns a new Logger that has this logger's context plus the given
+// context.
+func (el *ethLogger) New(ctx ...interface{}) log.Logger {
+	s := ""
+	for _, v := range ctx {
+		s = fmt.Sprintf("%s:%s", s, v)
+	}
+	l := el.dl.SubLogger(s)
+	return &ethLogger{dl: l}
+}
+
+// GetHandler gets the handler associated with the logger. Unused in geth. Does
+// nothing and returns a nil interface here.
+func (el *ethLogger) GetHandler() log.Handler {
+	return nil
```

Comment:
geth is always cool with `nil`?  Don't need a no-op function like `func(*Record) error { return nil }`?

Reply:
That would probably bet better. It is totally unused in the code at present, but that could change.

Reply:
Added the noop type. Was I able to do this with an anonymous function?

Reply:
dummyHandler looks nice